drivers/sensor/tmp108/tmp108.c

```diff
@@ -202,99 +192,106 @@ static int tmp108_attr_set(const struct device *dev,
 	uint16_t mode = 0;
 	uint16_t reg_value = 0;
 	int result = 0;
+	int32_t uval;
 
 	if (chan != SENSOR_CHAN_AMBIENT_TEMP && chan != SENSOR_CHAN_ALL) {
 		return -ENOTSUP;
 	}
 
 	switch ((int) attr) {
 	case SENSOR_ATTR_HYSTERESIS:
+		if (TI_TMP108_HYSTER_0_C(dev) == TI_TMP108_CONF_NA) {
+			LOG_WRN("AS621x Series lacks Hysterisis setttings");
```

Comment:
If sensor attribute is not supported, ,it should return -ENOTSUP instead a plain warning

Reply:
good catch, done